app/editor/src/features/admin/filters/FilterFormQuery.tsx

```diff
@@ -151,32 +152,23 @@ export const FilterFormQuery: React.FC = () => {
             </div>
           </Col>
         </Row>
-        <Row>
+        <Row gap="1rem">
           <label>Search for Keywords in: </label>
-          <Checkbox
-            id="chk_inHeadline"
-            checked={values.settings.inHeadline ?? false}
-            onChange={(e) => {
-              updateQuery('inHeadline', e.target.checked);
-            }}
+          <FormikCheckbox
```

Comment:
@CelloLiu We have FormikCheckbox and Checkbox.  Use them appropriately.  Also both support their own `label` property.